circle.yml

```diff
@@ -0,0 +1,21 @@
+# CircleCI configuration file.
+machine:
+  python:
+    version: 3.5.2
```

Comment:
Just noting for future reference that Python 3.6.1 is out (and available on Circle).